apps/meteor/server/startup/migrations/v310.ts

```diff
@@ -0,0 +1,27 @@
+import type { IPermission } from '@rocket.chat/core-typings';
+import { Permissions } from '@rocket.chat/models';
+
+import { upsertPermissions } from '../../../app/authorization/server/functions/upsertPermissions';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 308,
```

Comment:
```suggestion
	version: 310,
```

Reply:
I knew I'd forget to change it at least once :kekw:
thankss!